css-ui-4/Overview.bs

```diff
@@ -2166,81 +2166,91 @@ Document languages should specify
 for each <a>widget</a> that they define
 what needs to be preserved.
 
-The possible kinds of widgets are ''appearance/none'', ''button'', ''searchfield'', ''textfield'',
-''slider-horizontal'', ''checkbox'', ''radio'', ''listbox'', ''menulist'', ''menulist-button'',
-''textarea'', ''progress-bar'', ''meter''.
+The possible <dfn lt="kind of widget|kinds of widgets">kinds of widgets</dfn> are
+<dfn for="kind of widget" export>none</dfn>, <dfn for="kind of widget" export>button</dfn>,
+<dfn for="kind of widget" export>searchfield</dfn>, <dfn for="kind of widget" export>textfield</dfn>,
+<dfn for="kind of widget" export>slider-horizontal</dfn>, <dfn for="kind of widget" export>checkbox</dfn>,
+<dfn for="kind of widget" export>radio</dfn>, <dfn for="kind of widget" export>listbox</dfn>,
+<dfn for="kind of widget" export>menulist</dfn>, <dfn for="kind of widget" export>menulist-button</dfn>,
+<dfn for="kind of widget" export>textarea</dfn>, <dfn for="kind of widget" export>progress-bar</dfn>,
+<dfn for="kind of widget" export>meter</dfn>.
 
 <div algorithm="compute kind of widget">
-    UAs must run this at [=used value=] time: compute the kind of widget to use for an element, |element|:
+    UAs must run this at [=used value=] time: compute the [=kind of widget=] to use for an element, |element|:
 
     <ol>
         <!-- none always works -->
         <li>
-            If |computedAppearance| is ''appearance/none'', then return ''appearance: none''.
+            Let |computedAppearance| be the [=computed value=] of |element|'s 'appearance' property.
 
         <li>
-            Let |computedAppearance| be the [=computed value=] of |element|'s 'appearance' property.
+            If |computedAppearance| is [=kind of widget/none=], then return [=kind of widget/none=].
 
         <li>
-            Let |authorProps| be a map of all properties that have a <a>cascaded value</a> for
+            Let |authorProps| be a [=map=] of all properties that have a <a>cascaded value</a> for
             |element|, where the [=Author Origin=], [=Animation Origin=] and [=Transition Origin=]
             [=Cascade Origins=] are included and the [=User Origin=] and [=UA Origin=] [=cascade origins=]
             are ignored, where the key is the property name and the value is that <a>cascaded value</a>.
 
         <li>
-            If |element| can be a ''button'', given |computedAppearance|, then return the appropriate widget with
-            |authorProps| set to |authorProps|, |expected| set to button, and |fallback| set to
-            ''appearance/none''.
+            If |element| can be a button, given |computedAppearance|, then return the appropriate
```

Comment:
Adding a comment here so we don't lose track of it: "can be a button" (and similar below) should be links to a definition